### DISCUSSION TITLE
Avalonia extension to https://open-vsx.org/

### DISCUSSION THREAD

**Discussion by huuhaa** [AvaloniaUI/Avalonia#5873] [Ideas]
I'm not sure is this suggested before ( search for https://open-vsx.org/ found tons of issues , but none specific popped up at glanse). Never the less, I think it would be great if you could add Avalonnia extension also available at https://open-vsx.org/

If not, would be nice to hear some reasons behind it.

**Comment by rstm-sf** [comment]
There is no official extension for vscode ... there are only first sketches:
- https://github.com/kekekeks/Avalonia.Ide/
- https://github.com/ShadowDancer/PimpMyAvalonia
- https://github.com/rstm-sf/view-avalonia-preview-vscode

The latter is under development, but we can already see the first results by downloading the extension from [here](https://github.com/rstm-sf/view-avalonia-preview-vscode/actions/runs/756972290) in Artifacts and
```sh
code --install-extension view-avalonia-preview-0.0.1.vsix
```

https://user-images.githubusercontent.com/2669927/117037565-4ba2b080-ad0f-11eb-9944-d9e1d85bd1a9.mp4
